### PR TITLE
Banner Hide timeout

### PR DESCRIPTION
**Pull request by scotthasbrouck**

### CODE REVIEW

**Comment by dmitrig01** [comment]
I like this a lot – but I'm not sold on the name 'showAgain'

File: testbeds/web/example.html [hunk 1]
```diff
@@ -217,9 +217,10 @@
         description: 'Branch Demo app!',
         openAppButtonText: 'Open',
         downloadAppButtonText: 'Download',
-        iframe: true,     // Show banner in an iframe if CSS on your page is conflicting
-        showMobile: true, // true by default, just set here for an example
-        showDesktop: true // true by default, just set here for an example
+        iframe: true,      // Show banner in an iframe if CSS on your page is conflicting
+        showMobile: true,  // true by default, just set here for an example
+        showDesktop: true, // true by default, just set here for an example
+        showAgain: 1       // Don't show the user the banner again for 1 day
```
Comment (dmitrig01) [inline testbeds/web/example.html]: forgot to change back :P

File: src/3_branch.js [hunk 2]
```diff
@@ -1264,7 +1265,7 @@ if (WEB_BUILD) { // jshint undef:false
 			showAndroid: typeof options['showAndroid'] == 'undefined' ? true : options['showAndroid'],
 			showDesktop: typeof options['showDesktop'] == 'undefined' ? true : options['showDesktop'],
 			disableHide: !!options['disableHide'],
-			forgetHide: !!options['forgetHide'],
+			forgetHide: typeof options['forgetHideforgetHide'] == 'number' ? options['forgetHideforgetHide'] : !!options['forgetHideforgetHide'],
```
Comment (dmitrig01) [inline src/3_branch.js]: typ

Comment (dmitrig01) [inline src/3_branch.js]: o

Comment (scotthasbrouck) [inline src/3_branch.js]: oops. fixing now

**Comment by scotthasbrouck** [comment]
@dmitrig01 ok all fixed here

**Pull request merged**